installer/install.iss

```diff
@@ -1048,14 +1048,27 @@ end;
 
 function EnableSymlinksByDefault():Boolean;
 var
+    SymlinksForRegularUsers:Cardinal;
+    Root:Integer;
     ResultCode:Integer;
 begin
-    if IsOriginalUserAdmin then begin
-        Log('Symbolic link permission detection failed: running as admin');
-        Result:=False;
+    RegQueryDwordValue(HKEY_LOCAL_MACHINE, 'SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock', 'AllowDevelopmentWithoutDevLicense', SymlinksForRegularUsers)
+
+    if (SymlinksForRegularUsers = 1) then begin
+        // awesome, Developer mode enabled, we can create symlinks w/o being admin
+        Log('Symbolic links enabled, machine has Developer Mode enabled');
```

Comment:
I think we also need to verify that we're running on a "new enough" Windows 10: [Developer Mode only started allowing non-administrators to create symbolic links with Creators Update](https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/).

We should probably use [`GetWindowsVersionEx()`](http://www.jrsoftware.org/ishelp/index.php?topic=isxfunc_GetWindowsVersionEx) and then verify the Build:

```pascal
    Version:TWindowsVersion;
[...]
GetWindowsVersionEx(Version);
if ((SymlinksForRegularUsers=1) and (Version.Build>=14972)) then begin
[...]
```

Reply:
What's the policy on putting in fixes for unsupported versions of Windows? 

The first public release after `14972` is Windows 10 (1703) with build number 10.0.15063, and support for that one expired on October 8, 2019 for Enterprise (October 8, 2018 for regular editions), i.e. there's no supported Windows 10 version out there that doesn't have symlinks with Developer Mode. 

Happy to put in the fix if that's policy.

Reply:
At this point in the code, I believe it is uninteresting what Windows versions Git for Windows supports, and much more interesting which Windows build introduced the behavior on which your patch relies. Therefore using 14972 is the correct value to use here.